src/Google.Cloud.Functions.Invoker/EnvironmentVariablesConfigurationSource.cs

```diff
@@ -22,17 +22,17 @@ namespace Google.Cloud.Functions.Invoker
     /// An IConfigurationSource that deals with *specific* Functions Framework environment variables
     /// (unlike the more general EnvironmentVariablesConfigurationSource).
     /// </summary>
-    internal sealed class FunctionsEnvironmentVariablesConfigurationSource : IConfigurationSource
+    internal sealed class FunctionsEnvironmentVariablesConfigurationProvider : IConfigurationSource
```

Comment:
I'd rather keep calling this a source given that it implements IConfigurationSource instead of IConfigurationProvider.